Document and test direct Dockerfile usage without s2i scripts (#418)
sclorg/s2i-python-container: src/README.md

# Set the default command for the resulting image
CMD /usr/libexec/s2i/run
```

> A content of a Dockerfile without Source-to_image script would highly depends on your application and its needs.

Author: ```suggestion
Content of a Dockerfile without Source-to-image script would highly depend on your application and its needs.
```

Reviewer: ```suggestion
If you decide not to use the Source-to-Image scripts, you will need to manually tailor the Dockerfile to your application and its needs.
```

> An example for simple Django application might look like this:

Reviewer: ```suggestion
Example Dockerfile for a simple Django application:
```

```
FROM {{ spec.main_image }}

# Add application sources

> USER 0

Reviewer: Shouldn't we switch to `USER 1001` after copying and `chown`ing the sources, like in the previous example? AFAIK it's good practice not to run containers with `UID 0` inside, and we don't really need root, we can install the deps as `--user`. What do you think, @frenzymadness ? On the other hand, maybe it would be too much hassle to set up the priviledges?

Author: That makes perfect sense, thank you! I'm gonna implement it.

Author: I've realized that our images already have an active virtual environment so we don't have to install with `--user` but the switch to 1001 still makes sense.

Author: Or maybe not. From what I see in my local testing, switching user to 0 and back might not be necessary at all.
